.github/workflows/manual-build.yaml

```diff
@@ -0,0 +1,103 @@
+# This workflow is used to create git tags and releases with all needed artefacts automatically from a branch
+# If RELEASE is set to true, a git release will be created, with all needed artefacts.
+# If RELEASE is set to false, only build and push a docker image to the ECR with the commit hash
+name: CI Manual
```

Comment:
@vukomir @RuiSMagalhaes who can trigger this?
Any github user? if that is the case, can we restrict or disable?